server/src/main/java/org/opensearch/index/translog/transfer/TransferService.java

```diff
@@ -25,25 +28,40 @@ public interface TransferService {
 
     /**
      * Uploads the {@link TransferFileSnapshot} async, once the upload is complete the callback is invoked
-     * @param threadpoolName threadpool type which will be used to upload blobs asynchronously
+     * @param threadPoolName threadpool type which will be used to upload blobs asynchronously
      * @param fileSnapshot the file snapshot to upload
      * @param remotePath the remote path where upload should be made
      * @param listener the callback to be invoked once upload completes successfully/fails
      */
-    void uploadBlobAsync(
-        String threadpoolName,
+    void uploadBlobByThreadPool(
+        String threadPoolName,
```

Comment:
Please rename better

Reply:
There are two versions of upload. One via async IO and other via thread pool. To clarify that difference it was named like this. Keeping this in mind, anything which comes to your mind? 😄

Reply:
maybe just overload the method(same name)